### PR TITLE
Fixed issues in Drizzle and Contract Events guide

### LINKED ISSUE #1076: Fix Tutorial: Drizzle and Contract Events
[This tutorial](https://www.trufflesuite.com/guides/drizzle-and-contract-events) needs some updates. This issue came about as a result of work on trufflesuite/issue-target-tmp#1. 

*Issues to Fix*
- Fix links in blue boxes!
- Typo: “Processes each dispatched action~~s~~ before passing them to Reducers…”
- Need to be a bit more clear about where in the code certain code snippets go — assume complete lack of experience with react from the tutorial user
- Connecting to the `store` via this code is not working: 
```
// App.js
...
import store from './middleware'
...
<DrizzleProvider store={store} options={drizzleOptions}>
...
```

Overall this tutorial is pretty up to date! Shouldn't take long to make these small fixes.

### PR DESCRIPTION
**Pull request by dkillen**
Fixed issues in Drizzle and Contract Events guide reported in issue [#1076](https://github.com/trufflesuite/trufflesuite.com/issues/1076)

- Fixed broken links and typos.
- Added further description to ensure that it is clear to a user which files are being edited.
- Added examples of completed source code for comparison.
- Changed markdown heading levels so that table of contents displays on the sidebar.
- Removed unnecessary line breaks.

**Note:** The linked demo project now differs from the code produced by following the guide. The Issue #1076 reported a problem in the App.js file which is actually an issue in the demo project which lives in a different [repository](https://github.com/trufflesuite/drizzle-event-demo).

Resolves #1076

### CODE REVIEW

File: src/guides/drizzle-and-contract-events/index.md [hunk 1]
```diff
@@ -99,22 +88,17 @@ $ mkdir ./src/middleware
 $ touch ./src/middleware/index.js
 ```
 
-Import `EventActions` and `generateStore` from Drizzle as well as
+In `./app/src/middleware/index.js` import `EventActions` and `generateStore` from Drizzle as well as
```
Comment (fainashalts) [inline src/guides/drizzle-and-contract-events/index.md:91]:
```suggestion
In `./app/src/middleware/index.js`, import `EventActions` and `generateStore` from Drizzle as well as
```

File: src/guides/drizzle-and-contract-events/index.md [hunk 2]
```diff
@@ -144,24 +126,85 @@ export default generateStore({
...
 ...
 import store from './middleware'
 ...
 const drizzle = new Drizzle(drizzleOptions, store);
 ...
 ```
 
-Hook up Display
----------------
+You completed `App.js` file should looks something like this:
```
Comment (fainashalts) [inline src/guides/drizzle-and-contract-events/index.md:170]:
```suggestion
Your completed `App.js` file should looks something like this:
```

File: src/guides/drizzle-and-contract-events/index.md [hunk 3]
```diff
@@ -176,15 +219,74 @@ export default ({ accounts }) => (
...
+
+    </div>
+  );
+};
+
+```
+
+## A Quick Test
   * Things often go south during development so a pretest check is in order.
     1. MetaMask should NOT be on Main net! Do not run this if you're on main
```
Comment (fainashalts) [inline src/guides/drizzle-and-contract-events/index.md:285]:
```suggestion
    1. MetaMask should NOT be on Mainnet! Do not run this if you're on Mainnet!
```

File: src/guides/drizzle-and-contract-events/index.md [hunk 4]
```diff
@@ -176,15 +219,74 @@ export default ({ accounts }) => (
...
        net!
```
Comment (fainashalts) [inline src/guides/drizzle-and-contract-events/index.md:286]:
```suggestion
```

**Review by fainashalts — CHANGES_REQUESTED** [review]
Looks great! Had a couple tiny suggestions. Thanks!

**Comment by dkillen** [comment]
@cds-amal I have removed the leading $ from the cli commands.

**Review by cds-amal — APPROVED** [review]

**Comment by cds-amal** [comment]
Thanks @dkillen.

File: src/guides/drizzle-and-contract-events/index.md [hunk 5]
```diff
@@ -78,43 +70,35 @@ truffle(develop)> migrate
...
 import { toast } from 'react-toastify'
 ```
 
-The action `EventActions.EVENT_FIRED` is emitted whenever a contract event is
-detected in a Block. We will gain access to it by registering a custom
-middleware with the Redux store. As you know, Redux middleware comprises a set
-of functions executed in a sequence that processes each dispatched actions
-before passing them to Reducers.
-
+The action `EventActions.EVENT_FIRED` is emitted whenever a contract event is detected in a Block. We will gain access to it by registering a custom middleware with the Redux store. As you may know, Redux middleware comprises a set of functions, executed in a sequence, that process each dispatched action before passing it to Reducers. Add the following:
```
Comment (fainashalts) [inline src/guides/drizzle-and-contract-events/index.md:101]: I feel like Block doesn't need to be capitalized? But if it also capitalized elsewhere please ignore this!

```suggestion
The action `EventActions.EVENT_FIRED` is emitted whenever a contract event is detected in a block. We will gain access to it by registering a custom middleware with the Redux store. As you may know, Redux middleware comprises a set of functions, executed in a sequence, that process each dispatched action before passing it to Reducers. Add the following:
```

**Review by fainashalts — APPROVED** [review]
One small suggested change, otherwise this looks great to me! @cds-amal I see you had requested changes, I believe they have been addressed?

**Pull request merged**